### ISSUE TITLE
NO to toggle button  - macro to run at button release 

### ISSUE THREAD

**Issue by rbhun** [WhiteMagic/JoystickGremlin#268]
Hi

I need to take a button input on an Opencockpits axes cards, where I move a lever and it pushes a Normally Open button closed. It keeps it closed until the lever is forward. However in the sim the action required is a ‘toggle’. I need a function/macro that does a toggle when I open or close the switch. 

I have created a macro on the button, which presses a buoy button, waits, then depresses it. Together with Count (1) this works when I press the button, but I cannot
make the macro to run when the button is released.

**Comment by BenBlake79** [comment]
Two actions doing the same, one with "action condition" pressed and one with released...

**Comment by WhiteMagic** [comment]
Yep this sounds like you want the macro setup as you've done but then in the conditions tab add a condition on the action or container (depending how your setup looks like) and then add an "Action" condition with the condition of "released". This should result in your macro being run only once the physical input is released.
